.storybook/components/LocationSearch/LocationSearch.stories.tsx

```diff
@@ -63,7 +63,7 @@ const LocationSearchHeader = () => {
 }
 
 const LocationSearchScreen = () => {
-  const picker = useLocationSearchPicker({
+  const picker = useLocationsSearch({
```

Comment:
What encouraged the name change? Both to me sound alright

Reply:
`useLocationsSearch` is more general, also because the picker component is no longer the entry point component for the feature.